fares-data-build-tool/create-data
# fix: cfd-54 end date displays updated

# Description

-   End dates were being rounded up when stored in the database due to lack of milliseconds units in the database. This change updates the time we assign to end dates to avoid this rounding in future, however this won't fix display issues for previously made fares. To fix this, we've used the matching json endDate when displaying end date, as this hasn't been affected.

# Testing instructions

-   Create a point-to-point and a non-point-to-point ticket and ensure the end date is displayed correctly on /pointToPoinProducts and /otherProducts

# Type of change
